900.version-fixes/p.yaml

```diff
@@ -199,6 +199,8 @@
 - { name: prboom-plus,                                               ruleset: slackbuilds, untrusted: true } # accused of fkae 2.5.1.7um
 - { name: prelink,                     ver: "20151030",              ruleset: gentoo,      incorrect: true }
 - { name: prelink,                                                   ruleset: gentoo,      untrusted: true }
+- { name: primus,                      verpat: "20[0-9]{6}", incorrect: true } # several repos use a date, but this is a tagged release
```

Comment:
Having snapshots when the latest official release was half a decade ago should not be penalized by `incorrect` status.

Single `verpat: "20[0-9]{2}.*", snapshot: true` rule would be enough to ignore all dated versions, but it won't hurt to also mark `20150328` specifically as `incorrect` (since it's `0.2` judging by the date), and `20131226` as `outdated`.

Reply:
fixed with a538e06